source/MRMesh/MRObjectMeshHolder.h

```diff
@@ -119,9 +119,12 @@ class MRMESH_CLASS ObjectMeshHolder : public VisualObject
     ObjectMeshHolder( ProtectedStruct, const ObjectMeshHolder& obj ) : ObjectMeshHolder( obj )
     {}
 
-    const MeshTexture& getTexture() const { return texture_; }
-    virtual void setTexture( MeshTexture texture ) { texture_ = std::move( texture ); dirty_ |= DIRTY_TEXTURE; }
-    virtual void updateTexture( MeshTexture& updated ) { std::swap( texture_, updated ); dirty_ |= DIRTY_TEXTURE; }
```

Comment:
Probably we should keep old API for existing users, but mark it as [[deprecated]]?